### ISSUE TITLE
[Bug]: Alt Text on NFT Images should be trimmed while image is resolving

### ISSUE THREAD

**Issue by gambinish** [MetaMask/metamask-extension#26461]
### Describe the bug

When alt text gets rendered while NFT images are rendered (eg in cases where NFT src take a while to resolve, but alt text is available), the alt text for the image gets rendered in place of the image.

When the alt text is extremely long, it blows up the UI, making it hard to navigate the extension.

### Expected behavior

See if we can figure out a way to trim the alt text when an NFT fails to load, to make the asset visually appear nicer while asset src is resolving on smaller viewports? On larger screens (aka the detail view) this is less of an issue.

Sidenote that excessively long alt strings can have negative impact on accessibility:  https://equalizedigital.com/accessibility-checker/long-alternative-text/

Maybe there is a way to constrain them within a given boundary: https://stackoverflow.com/questions/2731484/can-i-wrap-img-alt-text

Another option I guess could be to render a loading indicator of some sort.

### Screenshots/Recordings

![Screenshot 2024-08-14 at 10 16 54 AM](https://github.com/user-attachments/assets/451fb108-c920-44f4-9298-4258ed56db81)

### Steps to reproduce

1. Go to `ui/components/multichain/nft-item/nft-item.js`
2. Set `src=""` to forcefully break the image
3. Set the `alt` to an extremely long string.

### Detection stage

In production (default)

### Version

7.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Severity

Low
